libs/frontend/util-command/src/lib/edit-styles-data.ts

```diff
@@ -1,23 +1,19 @@
 import { CommandType, ICommand, StyleType } from '@pubstudio/shared/type-command'
 import {
-  ICloseMixinMenuData,
   ICommandGroupData,
   ISetComponentCustomStyleData,
 } from '@pubstudio/shared/type-command-data'
 import { ISite } from '@pubstudio/shared/type-site'
 import { ref } from 'vue'
-import { getLastCommand, pushCommandObject, undoLastCommand } from './command'
-import { replaceLastCommand } from './replace-last-command'
+import { pushCommandObject, undoLastCommand } from './command'
+import { setEditingMixinData } from './editor-helpers'
 
 // Names of editing style properties
 export const editStyles = ref(new Set<string>())
 // Command group for actively editing styles
 export const editCommands = ref<ICommandGroupData | undefined>()
 export const currentStyleType = ref<StyleType>()
 
-// Extra info for editing reusable styles/mixins
-export const editingMixinData = ref<ICloseMixinMenuData | undefined>()
```

Comment:
I moved `editingMixinData` from here to the editor context so that mixin menu state (opened/closed) can be restored after refresh.